raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -1266,13 +1268,21 @@ contract TokenNetwork is Utils {
         internal
         returns (address signature_address)
     {
+        uint256 message_type_id = 1;
```

Comment:
Couldn't message types be an enum?

Reply:
Done. 
I type casted the `enum` to `uint256` before the encoding, to keep the padding to `32bytes`, otherwise it would have been tightly packed to `1byte`. As prevention for any malleability issues. `uint256(MessageTypeId.BalanceProof)`